bin/grax.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+./graxctl migrate 
+./graxctl secrets -force "{\"Provider\":\"aws\",\"Type\":\"storage\",\"Name\":\"StorageSecret\",\"Data\":{\"access_key_id\":\"$S3_AWS_ACCESS_KEY_ID\",\"provider\":\"aws\",\"region\":\"$S3_AWS_REGION\",\"s3_bucket\":\"$S3_BUCKET_NAME\",\"secret_access_key\":\"$S3_AWS_SECRET_ACCESS_KEY\"}}"
```

Comment:
This technically should check if the secret exists and only if not insert it. Otherwise it will clobber a bucket secret people enter in the web app after a dyno restart.

Reply:
Ok, we may need to think that through in the case of the add-on provider rolling the creds